### ISSUE TITLE
[20230305] Weekly AI ArXiv 만담 시즌2 - 8회차

### ISSUE THREAD

**Issue by scene-the-ella** [jungwoo-ha/WeeklyArxivTalk#74]

**Comment by jungwoo-ha** [comment]
### News
### ArXiv

**Comment by gyunggyung** [comment]
안녕하세요. 해당 스터디에 한 번도 참여 해본 적이 없고, 발표하려는 자료도 아직 반은 못 읽었습니다. 따라서 링크만 공유합니다.

그리고 질문이 있습니다.
1. 발표 시 얼굴이 나오지 않아도 되나요?
2. 저희 스터디가 9시 ~ 10시까지 진행하여, 저는 10시 이후에 참여하는 것도 괜찮을까요?
3. PPT나 별도의 자료 정리 없이, 논문이나 블로그만 보고 발표해도 되나요? 짧게 여기에 댓글 쓰는 정도의 자료 제작은 좋습니다.

괜찮다면 아래 자료들을 이야기 하고 싶습니다. 일부 제거 혹은 수정될 수도 있습니다. 자료를 보시면 제가 어떤 이야기를 하고 싶은지 아실 수도 있겠습니다. 모든 발표는 20분에서 40분 정도 걸릴 거 같습니다. 그리고 다음 주 일요일도 괜찮습니다! 감사합니다 :)

1. [Open source implementation for LLaMA-based ChatGPT training process. Faster and cheaper training than ChatGPT (wip)](https://github.com/nebuly-ai/nebullvm/tree/main/apps/accelerate/chatllama)
2. [Training a Helpful and Harmless Assistant with Reinforcement Learning from Human Feedback](https://arxiv.org/pdf/2204.05862.pdf)
3. [P-Tuning v2: Prompt Tuning Can Be Comparable to Fine-tuning Universally Across Scales and Tasks](https://arxiv.org/pdf/2110.07602.pdf)
4. [∞-former: Infinite Memory Transformer](https://arxiv.org/abs/2109.00301)
5. [AlexaTM 20B: Few-Shot Learning Using a Large-Scale Multilingual Seq2Seq Model](https://arxiv.org/pdf/2208.01448.pdf)
6. [Improving language models by retrieving from trillions of tokens](https://arxiv.org/pdf/2112.04426.pdf)

**Comment by veritas9872** [comment]
Consistency Models
ArXiv: https://arxiv.org/abs/2303.01469

Dropout Reduces Underfitting
ArXiv: https://arxiv.org/abs/2303.01500
GitHub: https://github.com/facebookresearch/dropout

Google USM: Scaling Automatic Speech Recognition Beyond 100 Languages
ArXiv: https://arxiv.org/abs/2303.01037

**Comment by ghlee3401** [comment]
## Audio, Speech, Music 
### Arxiv
- [WhisperX: Time-Accurate Speech Transcription of Long-Form Audio](https://arxiv.org/pdf/2303.00747.pdf)
    - **Keyword** : Voice Activity Detection (VAD)
    - **Code** : https://github.com/m-bain/whisperX

**Comment by jwlee-neubla** [comment]
- [Interactive Text Generation](https://arxiv.org/abs/2303.00908)
> - user simulator를 이용하여 interactive한 text generation 학습
> - generation model은 one-shot에 생성하기 때문에 hallucination과 같은 문제가 있어서 interactive한 방법이 중요함을 강조
<img width="488" alt="image" src="https://user-images.githubusercontent.com/93505462/222960612-4055e7db-f9c3-416a-af03-50bf1edd9e3c.png">
